storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -405,6 +429,14 @@ static my_bool get_one_option(int optid,
         extra_http_headers.insert(parse_http_header(argument));
       }
       break;
+    case OPT_CURL_RETRIABLE_ERRORS:
+      if (argument != nullptr) {
+        CURLcode code = static_cast<CURLcode>(atoi(argument));
+        if (code < CURLcode::CURL_LAST) {
+          curl_retriable_errors.push_back(code);
```

Comment:
@rahulmalik87 problem here is that we have not yet initialized the Http Client. It is a local object of main function. In order to implement it in the way you mentioned we need to:
1) Make it global
2) initialize it earlier in main (before we parse the arguments)

If you still think this will be a good idea, I can make the above changes.

Reply:
would be good to implement since it would avoid duplicate array.

Reply:
@rahulmalik87 Done